test/performance/dataplane-probe/main.go

```diff
@@ -48,12 +45,10 @@ func main() {
 	defer cancel()
 
 	// Use the benchmark key created
-	q, qclose, err := quickstore.NewAtAddress(ctx, &qpb.QuickstoreInput{
-		BenchmarkKey: proto.String(*benchmark),
-		Tags:         []string{"master"},
-	}, mako.SidecarAddress)
+	tags := []string{}
+	q, qclose, err := mako.SetupMako(ctx, *benchmark, tags)
```

Comment:
This will panic because you haven't infused `ctx` with the kubeclient.  See the `SetupInformers` call in load-test.

Reply:
moving this to load-test only. Will move it to common later, if needed in common.